caom2repo/caom2repo/core.py

```diff
@@ -131,12 +133,12 @@ def __init__(self, subject, logLevel=logging.INFO,
 
         self._repo_client = net.BaseWsClient(resource_id, subject,
                                              agent, retry=True, host=self.host,
-                                             idempotent_posts=True)
+                                             idempotent_posts=True, insecure=insecure)
         try:
             self._repo_client.caps.get_access_url(
                 CURRENT_CAOM2REPO_OBS_CAPABILITY_ID)
             self.capability_id = CURRENT_CAOM2REPO_OBS_CAPABILITY_ID
-            self.namespace = obs_reader_writer.CAOM24_NAMESPACE
+            self.namespace = obs_reader_writer.CAOM25_NAMESPACE
```

Comment:
Should the exception handle the CAOM24 namespace, or continue with the CAOM23 version?